test/conformance/api/v1/route_test.go

```diff
@@ -95,6 +95,52 @@ func getRouteURL(clients *test.Clients, names test.ResourceNames) (*url.URL, err
 	return url, err
 }
 
+// TestRouteGetAndList test ROUTE Get and LIST using Service as the only resource that we create, as Route CREATE is not required in the Spec
+func TestRouteGetAndList(t *testing.T) {
+	t.Parallel()
+	clients := test.Setup(t)
+
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   test.PizzaPlanet1,
+	}
+
+	// Clean up on test failure or interrupt
+	test.EnsureTearDown(t, clients, &names)
+
+	// Setup initial Service
+	if _, err := v1test.CreateServiceReady(t, clients, &names); err != nil {
+		t.Fatalf("Failed to create initial Service %v: %v", names.Service, err)
+	}
+
+	// Validate State after Creation
+	if err := validateControlPlane(t, clients, names, "1"); err != nil {
+		t.Error(err)
+	}
+
+	route, err := v1test.GetRoute(clients, names.Route)
+	if err != nil {
+		t.Fatal("Getting route failed")
+	}
+
+	routes, err := v1test.GetRoutes(clients)
```

Comment:
looks like this is already tested via the `checkRouteState` in [validateControlPlane](https://github.com/knative/serving/blob/ac4dc6a3c47565c427bd485ead9c0846319f9968/test/conformance/api/v1/util.go#L150). Thinking out loud I wonder if we really need an extra test for this, or whether we could just add a test of LISTing there too.

Reply:
@julz the problem with the reason why I've created a separate test is that checkRouteState can be used in other contexts when more things that are not required by the specs are used. I believe that having this separated with help with the filtering of which operations are used on each resource.

Reply:
is there a case where we want to check the route state but don't want to check it shows up in a LIST?

Reply:
I'd probably just drop the `validateControlPlane` check as it beyond the scope of the test